app-connector-client/server/app.ts

```diff
@@ -45,35 +43,31 @@ module.exports = function (varkesConfigPath = null, nodePortParam = null) {
     }
 
     app.use(expressWinston.logger(LOGGER))
-    app.use(express.static(path.resolve(__dirname, 'views/')))
+    app.set('view engine', 'ejs'); //* using EJS as template engine
+    app.set('views', path.join(__dirname, '/views/'));
+    app.use(express.static(path.resolve(__dirname, 'views/static/')))
 
-    app.use("/apis", apis)
+    app.use("/apis", apiRouter)
     app.use("/connection", connector) //* in the routes folder
 
-
-    app.get("/app", function (req, res) {
-        res.sendFile(path.resolve(__dirname, "views/index.html"))
+    app.get("/", function (_req, res) {
+        res.render('index', { appName: varkesConfig.name }) // read from varkesConfig defined above
     })
-    app.get("/metadata", function (req, res) {
+    app.get("/metadata", function (_req, res) {
         res.sendFile(path.resolve(__dirname, "resources/api.yaml"))
     })
-    app.get("/console", function (req, res) {
+    app.get("/console", function (_req, res) {
```

Comment:
Why it is renamed to "_api"? Is the underscore a typescript convention?

Reply:
this is for similar to go , suppress warnings that a variable is not being used.

Reply:
In go you can use the wildcard notation "_" like in "_, err := myFunction()", it is not a variable prefix.
I think the underscore prefix is historical from C to indicate a private variable, but even then it is in conflict with your usage scenario